flink-end-to-end-tests/test-scripts/docker-hadoop-secure-cluster/config/log4j.properties

```diff
@@ -0,0 +1,354 @@
+#log4j.appender.resourcemanagerrequestlog=org.apache.hadoop.http.HttpRequestLogAppender
+#log4j.appender.resourcemanagerrequestlog.Filename=${hadoop.log.dir}/jetty-resourcemanager-yyyy_mm_dd.log
+#log4j.appender.resourcemanagerrequestlog.RetainDays=3
+
+#log4j.logger.http.requests.jobhistory=INFO,jobhistoryrequestlog
+#log4j.appender.jobhistoryrequestlog=org.apache.hadoop.http.HttpRequestLogAppender
+#log4j.appender.jobhistoryrequestlog.Filename=${hadoop.log.dir}/jetty-jobhistory-yyyy_mm_dd.log
+#log4j.appender.jobhistoryrequestlog.RetainDays=3
+
+#log4j.logger.http.requests.nodemanager=INFO,nodemanagerrequestlog
+#log4j.appender.nodemanagerrequestlog=org.apache.hadoop.http.HttpRequestLogAppender
+#log4j.appender.nodemanagerrequestlog.Filename=${hadoop.log.dir}/jetty-nodemanager-yyyy_mm_dd.log
+#log4j.appender.nodemanagerrequestlog.RetainDays=3
+
+# Appender for viewing information for errors and warnings
+yarn.ewma.cleanupInterval=300
+yarn.ewma.messageAgeLimitSeconds=86400
+yarn.ewma.maxUniqueMessages=250
+log4j.appender.EWMA=org.apache.hadoop.yarn.util.Log4jWarningErrorMetricsAppender
+log4j.appender.EWMA.cleanupInterval=${yarn.ewma.cleanupInterval}
+log4j.appender.EWMA.messageAgeLimitSeconds=${yarn.ewma.messageAgeLimitSeconds}
+log4j.appender.EWMA.maxUniqueMessages=${yarn.ewma.maxUniqueMessages}
+
+## NameNode log
+log4j.appender.NAMENODE_RFA=org.apache.log4j.RollingFileAppender
+log4j.appender.NAMENODE_RFA.File=${hadoop.log.dir}/hadoop-namenode.log
+log4j.appender.NAMENODE_RFA.layout=org.apache.log4j.PatternLayout
+log4j.appender.NAMENODE_RFA.layout.ConversionPattern=%d{ISO8601} %p %c{2}: %m%n
+log4j.logger.org.apache.hadoop.hdfs.server.namenode=INFO,NAMENODE_RFA
+
+## DataNode log
+log4j.appender.DATANODE_RFA=org.apache.log4j.RollingFileAppender
+log4j.appender.DATANODE_RFA.File=${hadoop.log.dir}/hadoop-datanode.log
+log4j.appender.DATANODE_RFA.layout=org.apache.log4j.PatternLayout
+log4j.appender.DATANODE_RFA.layout.ConversionPattern=%d{ISO8601} %p %c{2}: %m%n
+log4j.logger.org.apache.hadoop.hdfs.server.datanode=INFO,DATANODE_RFA
+
+## ResourceManager log
+log4j.appender.RESOURCEMANAGER_RFA=org.apache.log4j.RollingFileAppender
+log4j.appender.RESOURCEMANAGER_RFA.File=${hadoop.log.dir}/hadoop-resourcemanager.log
+log4j.appender.RESOURCEMANAGER_RFA.layout=org.apache.log4j.PatternLayout
+log4j.appender.RESOURCEMANAGER_RFA.layout.ConversionPattern=%d{ISO8601} %p %c{2}: %m%n
+log4j.logger.org.apache.hadoop.yarn.server.resourcemanager=INFO,RESOURCEMANAGER_RFA
+
+## NodeManager log
+log4j.appender.NODEMANAGER_RFA=org.apache.log4j.RollingFileAppender
+log4j.appender.NODEMANAGER_RFA.File=${hadoop.log.dir}/hadoop-nodemanager.log
+log4j.appender.NODEMANAGER_RFA.layout=org.apache.log4j.PatternLayout
+log4j.appender.NODEMANAGER_RFA.layout.ConversionPattern=%d{ISO8601} %p %c{2}: %m%n
+log4j.logger.org.apache.hadoop.yarn.server.nodemanager=INFO,NODEMANAGER_RFA
+
+## HistoryServer log
+log4j.appender.HISTORYSERVER_RFA=org.apache.log4j.RollingFileAppender
```

Comment:
do we actually need all this?

Reply:
most of the stuff in there we don't really need. Removing them

Reply:
hmm, maybe they help in debugging. Leave them in just in case?